packages/core/src/fetchers/fetchAllEntities.ts

```diff
@@ -0,0 +1,96 @@
+import { ContentfulClientApi, Entry } from 'contentful';
+import { MinimalEntryCollection } from './gatherAutoFetchedReferentsFromIncludes';
+
+const MIN_FETCH_LIMIT = 1;
+
+const fetchEntities = async ({
+  entityType,
+  client,
+  query,
+}: {
+  entityType: 'Entry' | 'Asset';
+  client: ContentfulClientApi<undefined>;
+  query: { 'sys.id[in]': string[]; locale: string; limit: number; skip: number };
+}) => {
+  if (entityType === 'Asset') {
+    return client.getAssets({ ...query });
+  }
+
+  return client.withoutLinkResolution.getEntries({ ...query });
+};
+
+export const fetchAllEntities = async ({
+  client,
+  entityType,
+  ids,
+  locale,
+  skip = 0,
+  limit = 100,
+  responseItems = [],
```

Comment:
We're passing recursively `reponseItems[]` as a variable which is aggregator for all pages. However, as we don't pass recursively `includes{}`, I wonder if we're aggregating referenced entries recursively?